framework/kms-keyring.md

```diff
@@ -94,46 +99,35 @@ and MAY return a KMS Client that can make the following API calls in the given A
 If the client supplier cannot supply a client for the requested region,
 it MUST communicate that fact to the KMS keyring.
 
-### Key IDs
-
-Key IDs is a list of strings identifying KMS CMKs, in ARN format.
-This list identifies the CMKs to be used for data key encryption and decryption with this keyring.
+### Key Names
 
-Each Key ID MUST be one of the following:
+The key name identifies the [AWS KMS customer master keys (CMKs)](https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#key-id)
```

Comment:
[mixed plurality]
"key name" vs "identifies the CMKs that the keyring uses"

As the spec is currently defined, this is an input that accepts multiple values, each value being a key name.

Reply:
Good catch. Updating to "Key names identify..."